test/src/main/scala/scala/test/scala_import/BUILD

```diff
@@ -49,10 +49,23 @@ scala_specs2_junit_test(
     suffixes = ["Test"],
 )
 
+java_import(
+  name = "guava_and_commons_lang_java_import",
+  jars = ["@com_google_guava_guava_21_0//jar:file", "@org_apache_commons_commons_lang_3_5//jar:file"],
+)
+
 #Exports
 scala_import(
   name = "guava_and_commons_lang_as_exports",
-  exports = [":guava_and_commons_lang"],
+  exports = [":guava_and_commons_lang_java_import"],
```

Comment:
should we be changing this line? because i see you added an example below

Reply:
oops, I forgot to remote that during cleanup, let me push a commit